UnitsNet.Tests/BaseDimensionsTests.cs

```diff
@@ -733,13 +733,13 @@ public void CheckBaseDimensionMultiplicationWithForceEqualsMassTimesAcceleration
         [Fact]
         public void CheckToStringUsingMolarEntropy()
         {
-            Assert.Equal("[Length]^2[Mass][Time]^-2[Temperature]^-1[Amount]^-1", MolarEntropy.BaseDimensions.ToString());
+            Assert.Equal("[Length^2][Mass][Time^-2][Temperature^-1][Amount^-1]", MolarEntropy.BaseDimensions.ToString());
         }
 
         [Fact]
         public void CheckToStringUsingSpeed()
         {
-            Assert.Equal("[Length][Time]^-1", Speed.BaseDimensions.ToString());
+            Assert.Equal("[Length][Time^-1]", Speed.BaseDimensions.ToString());
```

Comment:
We haven't really discussed such change, but I personally find it slightly more readable. 
Not a strong preference, of course.. 😄

Reply:
Fine by me 👍